crypto/aes_openpgp.go

```diff
@@ -2,35 +2,68 @@ package crypto
 
 import (
 	"bytes"
-	"crypto"
-	"errors"
+	"fmt"
 	"io/ioutil"
-	"strings"
-
-	_ "crypto/sha256"
 
+	"github.com/bndw/pick/errors"
 	"golang.org/x/crypto/openpgp"
 	"golang.org/x/crypto/openpgp/armor"
 	"golang.org/x/crypto/openpgp/packet"
 )
 
 type AESOpenPGPClient struct {
+	config       Config
+	settings     AESOpenPGPSettings
 	packetConfig *packet.Config
 }
 
-func NewAESOpenPGPClient(config Config) (*AESOpenPGPClient, error) {
+type AESOpenPGPSettings struct {
+	Cipher   string `json:"cipher,omitempty" toml:"cipher"`
+	S2KCount int    `json:"s2kcount,omitempty" toml:"s2kcount"`
+}
 
-	// TODO(): Construct a packet from the config
-	// pc := &packet.Config{
-	// 	 DefaultHash: getHashFromConfig(config),
-	// }
+func DefaultAESOpenPGPSettings() *AESOpenPGPSettings {
+	return &AESOpenPGPSettings{
+		Cipher:   "aes256",
```

Comment:
https://github.com/bndw/pick/pull/48/commits/443bb7b466fcea3ceff70d6a440d57eed2378aea
and
https://github.com/bndw/pick/pull/48/commits/6a4c6f74c95653c5a9120485c632c24e44c3d6cf